<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mer::Internal::MerOptionsWidget</class>
 <widget class="QWidget" name="Mer::Internal::MerOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="sdkLabel">
           <property name="text">
            <string>Virtual Machine:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sdkComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Add...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startVirtualMachineButton">
           <property name="text">
            <string>Start Virtual Machine</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="launchSDKControlCenterPushButton">
           <property name="text">
            <string>Launch SDK Control Center</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="messageInfoLabel">
       <property name="text">
        <string>Please add a Virtual Machine running Mer to perform builds.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="Utils::DetailsWidget" name="sdkDetailsWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
       <widget class="QWidget" name="connectionInfoWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="connectionMessagesLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Connection messages:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="maximum">
              <number>0</number>
             </property>
             <property name="value">
              <number>-1</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="MessagesWindow" name="connectionMessagesWindow" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::DetailsWidget</class>
   <extends>QWidget</extends>
   <header location="global">utils/detailswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MessagesWindow</class>
   <extends>QWidget</extends>
   <header>messageswindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
